### ISSUE TITLE
Implement configuration of pazuzu

### ISSUE THREAD

**Issue by elgris** [zalando-incubator/pazuzu#69]
## User story

As a user of `pazuzu` I need ability to configure pazuzu application and its' parts.
## Example of config file

Please change any parts you think better to change. Format of config file may be changed as well.

Location: `~/.pazuzu`

```
[pazuzu]
  storageprovider=git

[git]
  url=https://github.com/Sangdol/pazuzu-test-repo/
  user=username
  password=password
```
## Acceptance criteria
- [x] Configuration of pazuzu is specified as a data structure in file `config.go`
- [x] Reading and writing configuration file is implemented in file `config.go`
- [x] Tests cover reading and writing the configuration.

**Comment by stoewer** [comment]
Two suggestions:
* If `~/.pazuzu` was a directory it could contain the configuration file as shown above (e.g. `~/.pazuzu/config` as well as other data such as locally cached metadata.
* Although I really like the .gitconfig like format of the configuration file, we should consider using yaml as format because we have a parser for yaml already in our dependencies.

**Comment by krapivchenkon** [comment]
Agree with Adrian regarding yaml format, it would be better

**Comment by jyun-zalando** [comment]
It seems almost implemented by #93
